From a1c69fc092755a59b25b7a90be3c97796bc77dc7 Mon Sep 17 00:00:00 2001
From: Romain Perier <romain.perier@collabora.com>
Date: Sun, 14 Aug 2022 04:29:32 +0000
Subject: [PATCH 098/120] WIP: ASoC: codecs: add support for ES8323

The ES8323 is compatible with the existing ES8328 codec driver, so add
compatibles allowing it to be used.

Signed-off-by: Romain Perier <romain.perier@collabora.com>
Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 sound/soc/codecs/es8328-i2c.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/codecs/es8328-i2c.c b/sound/soc/codecs/es8328-i2c.c
index 68072e99fcc7..57c7de5cfa05 100644
--- a/sound/soc/codecs/es8328-i2c.c
+++ b/sound/soc/codecs/es8328-i2c.c
@@ -16,6 +16,7 @@
 #include "es8328.h"
 
 static const struct i2c_device_id es8328_id[] = {
+	{ "es8323", 0 },
 	{ "es8328", 0 },
 	{ "es8388", 0 },
 	{ }
@@ -23,6 +24,7 @@ static const struct i2c_device_id es8328_id[] = {
 MODULE_DEVICE_TABLE(i2c, es8328_id);
 
 static const struct of_device_id es8328_of_match[] = {
+	{ .compatible = "everest,es8323", },
 	{ .compatible = "everest,es8328", },
 	{ .compatible = "everest,es8388", },
 	{ }
-- 
2.34.1

